src/Writer/Null.php

```diff
@@ -1,34 +0,0 @@
-<?php
-
-/**
- * @see       https://github.com/laminas/laminas-log for the canonical source repository
- * @copyright https://github.com/laminas/laminas-log/blob/master/COPYRIGHT.md
- * @license   https://github.com/laminas/laminas-log/blob/master/LICENSE.md New BSD License
- */
-
-namespace Laminas\Log\Writer;
-
-/**
- * Stub class for backwards compatibility.
- *
- * Since PHP 7 adds "null" as a reserved keyword, we can no longer have a class
- * named that and retain PHP 7 compatibility. The original class has been
- * renamed to "Noop", and this class is now an extension of it. It raises an
- * E_USER_DEPRECATED to warn users to migrate.
- *
- * @deprecated
- */
-class Null extends Noop
```

Comment:
I think deleting the class introduces a bc break. You shouldn't do that, unless the maintainers want to release a new major version (which the probably don't want to).

Reply:
In PHP 8  `Null` is a reserved keyword and produce an fatal error. https://3v4l.org/lKAQ4
Since its deprecated already i deleted this class. So yeah, yo are right. This is a BC break. I will update the PR description.

Reply:
Yes, it's already reserved since PHP7. But as long you don't use it, there is no problem.
But I don't know what's the bc break policy here, because if you drop PHP 5, the class will be useless anyway. maybe @boesing or @Xerkus know more?

Reply:
As the class is not instantiable in any of those versions we support within `composer.json`, deleting this class is **not** a BC break as it cannot be used in either of these versions anyway.